### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by SolarDrew**
`lightcurve` is dead. Long live `timeseries`.

### CODE REVIEW

**Comment by SolarDrew** [comment]
Oh, yes, it probably does. Please hold...

**Comment by SolarDrew** [comment]
Done that, plus removed all other references to lightcurve in the guide. I've replaced them all with timeseries, so @Alex-Ian-Hamilton and/or @Cadair might want to have a quick glance to check that those sentences are still valid.

File: docs/guide/acquiring_data/fido.rst [hunk 1]
```diff
@@ -63,6 +63,41 @@ logical ``OR`` operator would::
 The above query searches the Virtual Solar Observatory (VSO) showing
 that Fido can be used instead of explicitly searching the VSO.
 
+Indexing search results
+-----------------------
+
+The `UnifiedResponse` that Fido returns can be indexed to access a subset of
```
Comment (Cadair) [inline docs/guide/acquiring_data/fido.rst]: should link to the class like:

```
`~sunpy.net.fido_factory.UnifiedDownloader`
```

File: docs/guide/acquiring_data/fido.rst [hunk 2]
```diff
@@ -63,6 +63,41 @@ logical ``OR`` operator would::
...
+the search results. When doing this, the results should be treated as a
+two-dimensional array in which the first dimension corresponds to the clients
+which have returned results and the second is time.
```
Comment (Cadair) [inline docs/guide/acquiring_data/fido.rst]: 'records' not 'time', it might not be time (i.e. JSOC)

File: docs/guide/acquiring_data/fido.rst [hunk 3]
```diff
@@ -63,6 +63,41 @@ logical ``OR`` operator would::
...
+
+For example, the following code returns a response containing LYRA data from
+the `LYRAClient`, and EVE data from the `VSOClient`::
```
Comment (Cadair) [inline docs/guide/acquiring_data/fido.rst]: again, full path links to classes.

**Comment by nabobalis** [comment]
Ready to merge after #2170 is merged in.

**Pull request merged**